pyproject.toml

```diff
@@ -7,47 +11,52 @@ readme = "README.md"
 license = {file = "LICENSE"}
 dependencies = ['typing-extensions']
 classifiers = [
-    "Programming Language :: Rust",
-    "Programming Language :: Python :: Implementation :: CPython",
-    "Programming Language :: Python :: Implementation :: PyPy",
-    "License :: OSI Approved :: MIT License",
-    "Operating System :: POSIX :: Linux",
-    "Operating System :: MacOS",
-    "Operating System :: Microsoft :: Windows",
-    "Topic :: System :: Networking"
+  "Programming Language :: Rust",
+  "Programming Language :: Python :: Implementation :: CPython",
+  "Programming Language :: Python :: Implementation :: PyPy",
+  "License :: OSI Approved :: MIT License",
+  "Operating System :: POSIX :: Linux",
+  "Operating System :: MacOS",
+  "Operating System :: Microsoft :: Windows",
+  "Topic :: System :: Networking"
 ]
 authors = [
-    { name = "Samuel Yvon", email = "samuelyvon9@gmail.com" }
+  {name = "Samuel Yvon", email = "samuelyvon9@gmail.com"}
 ]
 maintainers = [
-    { name = "Samuel Yvon", email = "samuelyvon9@gmail.com" }
+  {name = "Samuel Yvon", email = "samuelyvon9@gmail.com"}
 ]
 
 [project.urls]
-homepage = "https://github.com/SamuelYvon/netifaces-2"
-issues = "https://github.com/SamuelYvon/netifaces-2/issues"
+Homepage = "https://github.com/SamuelYvon/netifaces-2"
+Issues = "https://github.com/SamuelYvon/netifaces-2/issues"
 
 [project.optional-dependencies]
 dev = [
-    "mypy >= 0.981",
-    "black >= 22.10.0",
-    "isort >= 5.10.1",
+  "mypy >= 0.991",
+  "black >= 22.12.0",
+  "ruff >= 0.0.238"
 ]
 
-[build-system]
-requires = ["maturin>=0.14,<0.15"]
-build-backend = "maturin"
+[tool.black]
+target-version = ["py311", "py310", "py39", "py38", "py37", "py36"]
 
 [tool.maturin]
 bindings = "pyo3"
 
-[tool.isort]
-profile = "black"
+[tool.mypy]
+python_version = "3.6"
+ignore_missing_imports = true
```

Comment:
> ignore_missing_imports = true

This one makes me a bit uncomfortable. With a `.mypy.ini` we could pin-point `netifaces.netifaces` as the package missing dependencies:

```
[mypy-netifaces.*]
ignore_missing_imports = True
```

Reply:
Right, I think that can be done with the overrides - https://mypy.readthedocs.io/en/stable/config_file.html#using-a-pyproject-toml-file

Reply:
Fair enough, I wasn't sure of the syntax within `pyproject.toml`

Reply:
like this?

Reply:
Syntax looks good to me, I'll check it out later (passes in CI tho)